### ISSUE TITLE
Maximising a ribbon window on Win7 "Non-Aero" Mode causes rendering issues  

### ISSUE THREAD

**Issue by robertmuehsig** [fluentribbon/Fluent.Ribbon#226]
I guess I found another WindowChrome bug: When the Window is maximised the desktop rendering is flickering and slow. In our application it is much worse then with the test app - we see a delay of a couple of seconds.

It only occures on Windows 7, in the "Non-Aero" Mode. We connect to the machine via Remote Desktop and I guess there is no hardware acceleration available. 

We are using .NET 4.0, but the problem occures also on .NET 4.5. The good news is: It seems with the dev branch the issue is fixed (https://github.com/fluentribbon/Fluent.Ribbon/issues/185) 

I made a short gif to demostrate the bug: 
![gif](https://cloud.githubusercontent.com/assets/756703/11926579/45629ff4-a7c6-11e5-9036-516c8b604339.gif)

Hope this helps. Any timeframe for a 4.0 "pre-release" nuget package?

**Comment by robertmuehsig** [comment]
Just made a small test run with our application and running against the dev 4.0 dlls: Works :+1: 

I noticed that with Fluent 4.0 the window has no shadow - is this by design or just a missing detail currently?
![image](https://cloud.githubusercontent.com/assets/756703/11926942/03c0ed50-a7c9-11e5-8ea5-11e4f94aff3b.png)

**Comment by batzen** [comment]
Pre-Release: Sometime this week.
Shadow: Will have a look.

**Comment by robertmuehsig** [comment]
Thanks!

**Comment by batzen** [comment]
Drop shadow get's killed by the WindowChrome class.
But I don't exactly know why and how to restore it...

**Comment by mezzalunaj** [comment]
The fix worked perfectly for me, thanks!! Just an FYI: when I referenced the 4.0.0 dll in my project, I started receiving a XamlParseException: Cannot set OverridesDefaultStyle property in the default Style exception. I tracked it back to the RibbonWindow.xaml file (line 695). I removed the _"OverridesDefaultStyle"_ setter and everything works great for me now. Just sharing in case someone else runs into the same issue. Cheers!! J.D.
